rocketpy/Rocket.py

```diff
@@ -649,18 +842,35 @@ def cnalfa1(cn):
                 extrapolation="natural",
             )
 
-            # Takes an approximation to an angular coefficient
-            clalpha = cldata.differentiate(x=0, dx=1e-2)
+        # Calculate Cd
+        if drag_coefficient is None and thickness is not None:
+            exposed_area = (1 / 2) * (rootChord + tipChord) * span
+            total_area = exposed_area + self.radius * rootChord
+
+            # In the formula, instead of span, it should be the distance between the
+            # midpoint of the root and the tip of the fin, but we are using
+            # span for now.
```

Comment:
If you consider a right-trapezoid, it should be `sqrt(span² + (Rc - tc)²/4)`. If you want to be more abstract, could get the angle bettwen span and rocket axis and use it on the law of cosines.

In my opinion, you don't need to make this correction on the code, I'm approving the pull request with the current approach. My point here is that it's not so hard to determine the distance you said, I think it's just about geometry. Reccomend save this idea for later.